packages/core/saved-objects/core-saved-objects-server/src/migration.ts

```diff
@@ -51,11 +51,6 @@ export type SavedObjectMigrationFn<InputAttributes = unknown, MigratedAttributes
 export interface SavedObjectsMigrationLogger {
   debug: (msg: string) => void;
   info: (msg: string) => void;
-  /**
-   * @deprecated Use `warn` instead.
-   * @removeBy 8.8.0
-   */
-  warning: (msg: string) => void;
```

Comment:
We should also adapt the implementation accordingly (`MigrationLogger` in https://github.com/elastic/kibana/blob/34c228b0c941308e01298e31223e7af0a5a2cee9/packages/core/saved-objects/core-saved-objects-migration-server-internal/src/core/migration_logger.ts#L19)

Reply:
Good catch, thanks!